docs/pages/dsCompositeIn/ds-compositeIn-L3-Low-Level-TestSpec.md

```diff
@@ -0,0 +1,196 @@
+# Device Settings Composite Input L3 Low Level Test Specification and Procedure Documentation
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Acronyms, Terms and Abbreviations](#acronyms-terms-and-abbreviations)
+- [References](#references)
+- [Level 3 Test cases High Level Overview](#level-3-test-cases-high-level-overview)
+- [Level 3 Python Test](#level-3-python-test-cases-high-level-overview)
+
+## Overview
+
+This document describes the L3 Low Level Test Specification and Procedure Documentation for the Device Settings Composite Input module.
+
+### Acronyms, Terms and Abbreviations
+
+- `HAL`    - Hardware Abstraction Layer
+- `API`    - Application Programming Interface
+- `L3`     - Level 3 Testing
+- `DS`     - Device Settings
+- `DUT`    - Device Under Test
+- `NA`     - Not Applicable
+- `RAFT`   - Rapid Automation Framework for Testing
+
+### References
+
+- `High Level Test Specification` - [ds-compositeIn-High-Level_TestSpec.md](https://github.com/rdkcentral/rdk-halif-test-device_settings/blob/3.1.4/docs/pages/ds-compositeIn-High-Level_TestSpec.md)
+
+- `HAL Interface file` - [dsCompositeIn Header 4.0.0](https://github.com/rdkcentral/rdk-halif-device_settings/blob/4.0.0/include/dsCompositeIn.h)
+
+
+## Level 3 Test Cases High Level Overview
+
+Below are top test use-case for the Composite Input.
+
+|#|Test-case|Description|Focus APIs|
+|-|---------|-----------|--------|
+|1|Verify the CompositeIn port connection with callbacks|Connect/disconnect the CompositeIn source device on each of compositeIn port and check the callbacks is triggered when the connection status changes|`dsCompositeInRegisterConnectCB()`|
+|2|Select the CompositeIn port |Select the composite input port and verify the status|`dsCompositeInSelectPort() dsCompositeInGetStatus()`|
+|3|Verify the CompositeIn active port status with callbacks|Select the CompositeIn port and check the callbacks is triggered when the active status changes|`dsCompositeInRegisterStatusChangeCB()`|
+|4|Scale the video and verify |Play a video in CompositeIn source device connected to the active CompositeIn port and scale the video resolution|`dsCompositeInScaleVideo()`|
```

Comment:
Do we need any particular stream to play , please specify the streams format and resolution.